### DISCUSSION TITLE
umlauts for new users who speak other languages

### DISCUSSION THREAD

**Discussion by VonderHeidFab** [netbox-community/netbox#15878] [Ideas]
Change the first line in the Postgres SQL setup manual to: CREATE DATABASE netbox WITH ENCODING 'UTF8'TEMPLATE=template0;
so that umlauts in German are taken into account and no error message is generated.

**Comment by markkuleinio** [comment]
Can you be more specific? What error message are you receiving and in which phase?

I have lots of NetBox users that have umlauts (including myself) and I have no error messages in NetBox.

Markku Leiniö

**Comment by VonderHeidFab** [comment]
In our Netbox installation, the database is encoded using standard ASCII,
which is not a problem for the installation. But as soon as you type in an
umlaut äöü or ß and save the process, Django reports a database error that
the database cannot do anything with the character and therefore cannot
save it. I therefore suggest setting up the database using UTF8 as
standard, as the required characters are available there.

Markku Leiniö ***@***.***> schrieb am Mo., 29. Apr. 2024,
06:46:

**Reply by kkthxbye-code** [reply]
> I therefore suggest setting up the database using UTF8 as
standard, as the required characters are available there.

This is the default on all modern Linux installs. Postgres gets the default encoding from LC_CTYPE, so it's uncommon to experience the issue you did.
